src/families/osmosis/specs.ts

```diff
@@ -0,0 +1,69 @@
+import invariant from "invariant";
```

Comment:
Nice adding this one ! we will make some run to try if it's work

Reply:
thanks, it's used in the same way as Celo here: https://github.com/LedgerHQ/ledger-live-common/blob/bcd322ccf210e13a45e75692e898c58fa73fa535/src/families/celo/specs.ts#L1

Reply:
Henri was mentioning the whole file I guess, which allows to run our automated testing tool that we call "the bot", and I agree thanks for providing it :+1: :slightly_smiling_face:

Reply:
ohhh got it, no problem at all!